### DISCUSSION TITLE
Add Import and sort categories

### DISCUSSION THREAD

**Discussion by pcwizard7** [X2CommunityCore/xcom2-launcher#270] [Ideas]
Could you add the ability to import the workshop steam mod categories? Then auto sort the mods to each category for better management.

Thanks

**Comment by RevZero** [comment]
Are you referring to the workshop tags?
You can use/add those tags from AML by right clicking the desired mod(s) and selecting "Use workshop tags". And having them somehow sorted/grouped by tags wouldn't be useful, because there are so many different tags.
